Lib/fractions.py

```diff
@@ -90,6 +90,13 @@ def __new__(cls, numerator=0, denominator=None, *, _normalize=True):
         Fraction(147, 100)
 
         """
+        # private _normalize=False should only be set if the Fraction is
```

Comment:
Let's drop this comment for this PR, and keep the PR focused on the `limit_denominator` changes. We could open a separate PR for this change.